firstvoices/backend/views/site_feature_views.py

```diff
@@ -109,3 +118,31 @@ def get_queryset(self):
             "created_by",
             "last_modified_by",
         )
+
+    def perform_create(self, serializer):
+        # Once a site feature is created via the API, sync all media content in indexes
+        instance = serializer.save()
+        site = instance.site
+
+        # Queue the site media sync after updating the model
+        transaction.on_commit(
+            lambda: sync_all_media_site_content_in_indexes.apply_async(
+                (site.id,), link_error=link_error_handler.s()
+            )
+        )
```

Comment:
I know we're only doing this twice, but I think it is low-level enough that we extract it into a function called something like request_sync_media_index_for_site . You could put that function in search.tasks.site_content_indexing_tasks -- we have similar ones in the index_manager_tasks module.